src/formElementPlugins/pluginProvider.tsx

```diff
@@ -62,7 +67,8 @@ class pluginProvider {
 }
 
 function getLocalElementPlugin(folderName: string) {
-  const result: PluginComponents = {}
+  const result: PluginComponents = {} as PluginComponents
+  result.config = require(`./${folderName}/pluginConfig.json`)
```

Comment:
Will rethink this when making plugins work on prod

Reply:
Yes, for sure. I wasn't particularly happy with it, but didn't see a straightforward way to return the config info the the Wrapper without a significant refactoring of the pluginProvider.